client/my-sites/people/people-list-item/style.scss

```diff
@@ -34,33 +34,6 @@
 	}
 }
 
-.people-list-item__checkbox[type=checkbox] {
-	margin-top: 0;
-	position: absolute;
-	top: 48px;
-	left: 2px;
-
-	&::after { // Making tap area larger
-		content: '';
-		position: absolute;
-		top: -20px;
-		left: -19px;
-		width: 56px;
-		height: 55px;
-	}
-}
-
-.people-list-item__label {
```

Comment:
The `.people-list-item__label` also has a record in the global z-index map. Worth removing, too.

Reply:
Good eye, I have removed this in d906c1338401cfa486cf8831e76f51f1c52c9297.